v2.0/src/Components/Pages/ServicesPage/ServicesPage.js

```diff
@@ -50,7 +50,7 @@ class ServicesPage extends React.Component {
         return vendors.map((vendor) => {
             return (
                 <div className="col-12 col-md-6 col-lg-4" key={vendor.vendor}>
-                    <div className="card rounded-0">
+                    <div className="card rounded-0 spacing">
```

Comment:
I think uploading a picture of the page will make it easier to see how things look with these style specific changes